File: perf.js
const { performance } = require("perf_hooks");

const start = performance.now();
const size = 65535;
let answer = 0;

for(let i = 1; i <= size; i++) {
  for(let j = 1; j <= size; j++) {
    if(i % j === 0) answer++;
  }
}

const diff = performance.now() - start;
console.log(answer.toString());
console.log(diff / 1000 + "s");

File: perf.java
class perf {
  public static void main(String[] args) {
    int answer = 0;
    long start = System.currentTimeMillis();
    for(int i = 1; i <= 65535; i++) {
      for(int j = 1; j <= 65535; j++) {
        if(i % j == 0) answer++;
      }
    }
    System.out.println(answer);
    System.out.println(Float.toString((System.currentTimeMillis() - start) / 1000f) + "s");
  }
}
